upgrade_order.md

```diff
@@ -0,0 +1,6 @@
+LocalGroupIndex -> Community -> Website -> User
```

Comment:
LocalUserIndex is missing despite having a CHANGELOG entry. Is that because it can go in any order?

Reply:
LocalUserIndex + GroupIndex can go whenever